core/domain/rights_manager.py

```diff
@@ -512,16 +851,35 @@ def can_unpublicize(self, activity_type, activity_id):
 
 def _assign_role(
         committer_id, assignee_id, new_role, activity_id, activity_type):
-    """Args:
-    - committer_id: str. The user_id of the user who is performing the action.
-    - activity_rights: The storage object for the rights of this activity (
-        one of: ExplorationRightsModel or CollectionRightsModel).
-    - assignee_id: str. The user_id of the user whose role is being changed.
-    - new_role: str. The name of the new role: either 'owner', 'editor' or
-        'viewer'.
-    - activity_id: str. The ID of the exploration or collection.
-    - activity_type: str. One of feconf.ACTIVITY_TYPE_EXPLORATION or
-        feconf.ACTIVITY_TYPE_COLLECTION.
+    """Assign a new role to the user.
+
+    Args:
+        committer_id: str. ID of user who is performing the action.
+        activity_rights: The storage object for the rights of this activity.
```

Comment:
activity_rights: ExplorationRightsModel|CollectionRightsModel. The storage object for the rights of this activity.